code/Control/UserDefinedFormController.php

```diff
@@ -672,4 +678,20 @@ function (){
 
         return $result;
     }
+
+    /**
+     * Check validity of email and return array of valid emails
+     */
+    protected function validEmailsToArray(?string $emails): array
+    {
+        $emailsArray = [];
+        $emails = explode(',', $emails ?? '');
+        foreach ($emails as $email) {
+            if (filter_var($email = trim($email), FILTER_VALIDATE_EMAIL)) {
```

Comment:
Use `Egulias\EmailValidator\EmailValidator` instead (already installed as a dependency) as it's the same validator that will be used later by symfony/mime to validate the email addresses, though if they fail validation later on they'll throw an exception, rather than tolerating user error such as what they're doing here

Reply:
See comment https://github.com/silverstripe/silverstripe-userforms/pull/1208#discussion_r1195818508

Reply:
https://github.com/silverstripe/silverstripe-userforms/pull/1208#discussion_r1195810023
See comment https://github.com/silverstripe/silverstripe-userforms/pull/1208#discussion_r1195800807

Reply:
DONE